### DISCUSSION TITLE
Uppermost menu buttons can no longer be accessed by moving pointer all the way up

### DISCUSSION THREAD

**Discussion by ghost** [ppy/osu#26214] [Ideas] [answered]
I think this is a regression as of the most recent update 2023.1229.0.

If I have my pointer at the very top of the screen:
![image](https://github.com/ppy/osu/assets/35064754/a30e50d6-4965-4f6f-adb7-e2999a94c129)

If I just move it a few pixels below:
![image](https://github.com/ppy/osu/assets/35064754/5e5fa99a-e48a-44db-847f-3a6cf67e3dd8)

Before these changes to these buttons I could reach all these buttons by having my pointer at the very top of the screen. I thought it was very ergonomic.

This applies especially to the settings and notifications buttons in the top left and top right, respectively. I could previously quickly open my notifications by just throwing my pointer to the very upper right area of my screen without thinking much. Now I have to move my pointer a few pixels down toward the bottom right. This might seem insignificant but in my case my tablet area isn't the full tablet's area (to adjust sensitivity) and so it actually takes longer than you would think to move it just a few pixels down to the actual position of the settings button.
It's just a little annoying.

Basically, it'd be nice if it was the same as the buttons at the bottom of the screen where this behavior is still present (the random button etc.)

**Comment by ghost** [comment]
I just noticed that this behavior still seems to exist but only for the 4 osu! mode setting buttons in the top left. If you have your pointer at the very top you can select osu!taiko for example without the actual button being in the "hovered state" (transparent white rounded rectangle appears).

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu/discussions/26198
